crates/ruff_python_parser/src/snapshots/ruff_python_parser__lexer__tests__numbers.snap

```diff
@@ -79,7 +79,7 @@ expression: lex_source(source)
     ),
     (
         Int {
-            value: 0x995DC9BBDF1939FA,
+            value: 11051210869376104954,
```

Comment:
We should update the test. I think this test intentionally captures the case where the `Number` is `Big`.